packages/router/src/Route.php

```diff
@@ -20,4 +20,9 @@ interface Route
     public array $middleware {
         get;
     }
+
+    /** @var class-string<HttpMiddleware>[]  */
+    public array $without {
```

Comment:
Technically this is a breaking change. Since we intend on tagging 2.x soon, I think it's best to move this to 2.x